src/openqaoa-qiskit/tests/test_gate_applicators_qiskit.py

```diff
@@ -3,6 +3,8 @@
 from qiskit.circuit import gate as qsk_gate
 from qiskit.circuit import controlledgate as qsk_c_gate
 from qiskit.circuit.library import standard_gates as qsk_s_gate
+
+# from qiskit.circuit import singleton as qsk_sgt
```

Comment:
The XGate was displaced to qiskit.circuit.Singleton. All non-parametrized gates have been changed to Singleton objects and passed only by reference to the overarching circuit.
It was later resolved (I'm assuming that it's now accessible through qiskit.circuit.gate) and therefore left it as a comment.

Reply:
so, the previous imports work again now?

Reply:
I have already approved the PR, we can merge it once we resolve this conversation

Reply:
The error was that the XGate (unparametrized) had been moved from `qiskit.circuit.gate` to `qiskit.circuit.singleton`. Which apparently was later changed such that the tests would pass again. I ran the tests again to make sure that they still function correctly.